scripts/loadMemberImages.js

```diff
@@ -0,0 +1,32 @@
+import { readFileSync } from "fs";
+import { promisify } from "util";
+import { exec } from "child_process";
+
+const execPromise = promisify(exec);
+
+async function run() {
+  await execPromise("mkdir -p build/members");
+
+  const fileContents = readFileSync("build/server/members.json");
+  const members = JSON.parse(fileContents.toString());
+  const allMembers = members.active.concat(members.alumni);
+
+  await Promise.all(allMembers.map(loadMemberImage));
+}
+
+async function loadMemberImage(member) {
+  const url = `${member.picture.src}?h=1000&fmt=jpg`.replace("////", "//");
+  const nameSnakeCase = member.name
+    .normalize("NFD")
+    .replace(/\p{Diacritic}/gu, "")
+    .toLowerCase()
+    .replace(/ /g, "_")
+    .replace(/-/g, "");
+
+  const filePath = `build/members/${nameSnakeCase}.jpg`;
+
+  await promisify(exec)(`curl -o ${filePath} ${url}`);
+  console.log(`Loaded ${filePath}`);
```

Comment:
is there a reason this was left in?

Reply:
Yeah, figure it would be useful to see the output state. We already have a lot of busy output from the rest of the build script so I figured adding this wouldn't be too bad.